README.md

```diff
@@ -1,73 +1,44 @@
 ![Build Status](https://storage.googleapis.com/cloud-tools-for-java-kokoro-build-badges/intellij-ubuntu-master-previous-version-orb.svg)
 ![Build Status](https://storage.googleapis.com/cloud-tools-for-java-kokoro-build-badges/intellij-ubuntu-master-eap-orb.svg)
 
-The plugin integrates the [Google Cloud Platform](https://cloud.google.com/)
-into the IntelliJ IDEA UI. Currently this includes:
+Cloud Tools for IntelliJ is a plugin that helps facilitates cloud development. This includes the 
+following areas:
+
+* [Kubernetes](https://github.com/GoogleCloudPlatform/google-cloud-intellij/tree/master/kubernetes)
+* Google Cloud Platform
+  * [Google Cloud Source Repositories](https://cloud.google.com/tools/cloud-repositories/) 
+    Fully-featured private Git repositories hosted on Google Cloud Platform.
+  * The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/) 
+    The Cloud Debugger can inspect the state of a Java application running on 
+    [Google App Engine](https://cloud.google.com/appengine/)
+    at any code location without stopping the application.
+  * [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
+  * [Google Cloud Storage](https://cloud.google.com/storage/) 
+    Browse your Google Cloud Storage buckets.
+  * [Google Cloud Java Client Libraries](https://cloud.google.com/tools/intellij/docs/client-libraries) 
+    Add Java client libraries to your project, enable Google Cloud APIs, and create service accounts.
+
+    (_For detailed user documentation on GCP features, visit our documentation_
+ [website](https://cloud.google.com/tools/intellij/docs/?utm_source=github&utm_medium=google-cloud-intellij&utm_campaign=ToolsforIntelliJ)).
 
-* [Google Cloud Source Repositories](https://cloud.google.com/tools/cloud-repositories/) 
-  Fully-featured private Git repositories hosted on Google Cloud Platform.
-* The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/) 
-  The Cloud Debugger can inspect the state of a Java application running on 
-  [Google App Engine](https://cloud.google.com/appengine/)
-  at any code location without stopping the application.
-* [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
-* [Google Cloud Storage](https://cloud.google.com/storage/) 
-  Browse your Google Cloud Storage buckets.
-* [Google Cloud Java Client Libraries](https://cloud.google.com/tools/intellij/docs/client-libraries) 
-  Add Java client libraries to your project, enable Google Cloud APIs, and create service accounts.
+## Supported Platforms
 
-For detailed user documentation go to our documentation
- [website](https://cloud.google.com/tools/intellij/docs/?utm_source=github&utm_medium=google-cloud-intellij&utm_campaign=ToolsforIntelliJ).
+The Cloud Tools for IntelliJ plugin supports the entire JetBrains family of IDEs, versions 2018.2 or 
```

Comment:
Could it be beneficial to move this section to the top, first sentence where we talk about plugin for cloud development, before feature set?

Reply:
I know we have *Supported Platforms* after that, but initial sentence strongly hints towards IntelliJ, which is Java only.

Reply:
good idea. I sort of duplicated this notion at the top to make it clearer. let me know if this helps